src/i9n/fungible.rs

```diff
@@ -49,6 +50,27 @@ impl Runtime {
         Ok(reply)
     }
 
+    pub fn accept(
```

Comment:
Can we pls move this right before `validate` command, not before the `issue`?

Reply:
sure, done